src/commands/slash/suggestion.ts

```diff
@@ -0,0 +1,120 @@
+import { SlashCommand } from "../../utils/SlashCommand";
+import { Logger } from "../../utils/Logger";
+import {
+	ApplicationCommandType,
+	ApplicationCommandOptionType,
+} from "discord-api-types/payloads";
```

Comment:
How many times I'll correct you this, always check typescript auto-imports
```suggestion
} from "discord-api-types/v9";
```

Reply:
:skull: